Makefile_i8_PaStiX

```diff
@@ -0,0 +1,133 @@
+# See our wiki for getting the CalculiX dependencies:
```

Comment:
Maybe you want to base this `Makefile` to the one from `master` as well?
Also, look out for any references to the wiki, which now need to point to the adapter documentation on the website.

Reply:
I'm not sure I follow, this is already a modified version of `master` makefile. 
Agree for the wiki, I didn't really pay attention to this at first.

Reply:
Ah, right, I was confused for a moment.

Reply:
Keeping this unresolved only to remember to update according to the latest `Makefile`.

By the way, instead of maintaining two Makefiles, could we maybe add a separate target in the same Makefile?

Reply:
Probably, I can look into that